index.html

```diff
@@ -147,6 +147,9 @@ <h2>
         [Exposed=Window]
         interface Gamepad {
           readonly attribute DOMString id;
+          readonly attribute unsigned short? vendorId;
+          readonly attribute unsigned short? productId;
```

Comment:
Same as above?

Reply:
We should also provide some guidance here... right now, for instance, Firefox and Chrome are different in their IDs with respect to the "0":

Chrome is:
"DualSense Wireless Controller (STANDARD GAMEPAD Vendor: 054c Product: 0ce6)"
Firefox is:
"54c-ce6-DualSense Wireless Controller"

It might make them easier to identify?

Reply:
Hmm... I guess it's just as easy to do:

```JS
gamepad.productId?.toString(16);
```

Reply:
Actually, yeah... forget it. Let's do unsigned short is fine.